src/Build/Utilities/EngineFileUtilities.cs

```diff
@@ -225,9 +225,14 @@ private static bool MatchesLazyWildcard(string fileSpec)
             return _regexMatchCache.Value.GetOrAdd(fileSpec, file => s_lazyWildCardExpansionRegexes.Any(regex => regex.IsMatch(fileSpec)));
         }
 
-        /// Returns a Func that will return true IFF its argument matches any of the specified filespecs
-        /// Assumes filespec may be escaped, so it unescapes it
+        /// <summary>
+        /// Returns a Func that will return true IFF its argument matches any of the specified filespecs.
+        /// Assumes filespec may be escaped, so it unescapes it.
         /// The returned function makes no escaping assumptions or escaping operations. Its callers should control escaping.
+        /// </summary>
+        /// <param name="filespecsEscaped"></param>
+        /// <param name="currentDirectory"></param>
```

Comment:
Technically should be something here, but these names were pretty clear in my opinion.

Reply:
I think this will be covered by another analyzer. Not 100% sure though.